### PR TITLE
Test pull

### PR DESCRIPTION
**Pull request by ArpadK**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -59,7 +61,10 @@ public void paintComponent(Graphics graphics) {
             super.paintComponent(graphics);
 
             //System.out.println("* Repaint *");
-
+            String probeersel = "joost";
+            if(probeersel == "joost"){
```
Comment (aarsweiland) [inline src/Main.java]: MAJOR Either remove or fill this block of code. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS00108)

File: src/Main.java [hunk 2]
```diff
@@ -13,7 +13,9 @@
 //TODO known issue - sometimes ball gets stuck behind human paddle
 
 public class Main {
+    // code aanpassing
 
+    public int testi = 0;
```
Comment (ArpadK) [inline src/Main.java:18]: MINOR Make testi a static final constant or non-public and provide accessors if needed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)

File: src/Main.java [hunk 3]
```diff
@@ -13,7 +13,9 @@
 //TODO known issue - sometimes ball gets stuck behind human paddle
```
Comment (ArpadK) [inline src/Main.java:13]: INFO Complete the task associated to this TODO comment. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)

File: src/Main.java [hunk 4]
```diff
@@ -59,7 +61,10 @@ public void paintComponent(Graphics graphics) {
             super.paintComponent(graphics);
 
             //System.out.println("* Repaint *");
```
Comment (ArpadK) [inline src/Main.java:63]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by aarsweiland** [comment]
SonarQube analysis reported 19 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 4 critical
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [Main.java#L101](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L101): Remove this call to "exit" or ensure it is really required. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1147)
1. ![CRITICAL][CRITICAL] [Main.java#L106](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L106): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [Main.java#L111](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L111): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Main.java#L130](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L130): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Main.java#L137](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L137): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [Main.java#L46](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L46): This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Main.java#L114](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L114): Replace this usage of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Main.java#L118](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L118): Replace this usage of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Main.java#L166](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java#L166): Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [Main.java](https://github.com/aarsweiland/PingPong/blob/15372f8649a53f3e671e5ede044f510d2c1c99a8/src/Main.java): Move this file to a named package. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1220)

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: src/Main.java [hunk 5]
```diff
@@ -59,7 +61,10 @@ public void paintComponent(Graphics graphics) {
...
-
+            String probeersel = "joost";
+            if(probeersel == "joost"){
+                //hooooofoft;
```
Comment (ArpadK) [inline src/Main.java:66]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by ArpadK** [comment]
SonarQube analysis reported 19 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 4 critical
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [Main.java#L101](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L101): Remove this call to "exit" or ensure it is really required. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1147)
1. ![CRITICAL][CRITICAL] [Main.java#L106](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L106): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [Main.java#L111](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L111): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Main.java#L130](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L130): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Main.java#L137](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L137): Make the enclosing method "static" or remove this set. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [Main.java#L46](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L46): This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Main.java#L114](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L114): Replace this usage of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Main.java#L118](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L118): Replace this usage of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Main.java#L166](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java#L166): Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [Main.java](https://github.com/aarsweiland/PingPong/blob/b31dc4eb068149069050c640efb000bf53ff4373/src/Main.java): Move this file to a named package. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1220)

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**